bun.lock

```diff
@@ -15,6 +14,7 @@
         "braintrust": "*",
         "changelogithub": "^13.15.0",
         "glob": "^11.0.3",
+        "@biomejs/biome": "https://pkg.pr.new/biomejs/biome/@biomejs/biome@a6874f9",
```

Comment:
How long until official version?

Reply:
no clue. will ask in the thread.
other option is to downgrade to 2.0.4 or 2.0.6.
the thread says 2.0.6 is stable. our monorepo is pinned on 2.0.4 and is def stable

Reply:
https://github.com/biomejs/biome/issues/6784#issuecomment-3082022887

Reply:
@sam-goodwin 
> We generally don't provide ETAs, but I think its safe to say that at least a patch release will be soon.